safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -117,6 +117,12 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         1,
         IntervalSchedule.HOURS,
     ),
+    CeleryTaskConfiguration(
+        "safe_transaction_service.analytics.tasks.get_transactions_per_safe_app_task",
+        "Run query to get number of transactions grouped by safe-app",
+        1,
+        IntervalSchedule.DAYS,
```

Comment:
This should be 1 week.

Reply:
Done [here](https://github.com/safe-global/safe-transaction-service/pull/1177/commits/6cf257018140f2f7f87f7c496964f9d0889be999)